experimental/web_dashboard/tool/grind.dart

```diff
@@ -0,0 +1,106 @@
+// Copyright 2020, the Flutter project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+import 'dart:convert';
+import 'dart:io';
+import 'package:path/path.dart' as path;
+
+import 'package:grinder/grinder.dart';
+
+main(args) => grind(args);
+
+@Task()
+void runSkia() {
+  run('flutter',
+      arguments:
+          'run -d web --web-port=5000 --release --dart-define=FLUTTER_WEB_USE_SKIA=true lib/main.dart '
+              .split(' '));
+}
+
+@Task()
+void runWeb() {
+  run('flutter',
+      arguments: 'run -d web --web-port=5000 lib/main.dart '.split(' '));
+}
+
+@Task()
+void runMock() {
+  run('flutter',
+      arguments: 'run -d web --web-port=5000 lib/main_mock.dart '.split(' '));
+}
+
+@Task()
+void runMockSkia() {
+  run('flutter',
+      arguments:
+          'run -d web --web-port=5000 --release --dart-define=FLUTTER_WEB_USE_SKIA=true lib/main_mock.dart'
+              .split(' '));
+}
+
+@Task()
+test() {
+  TestRunner().testAsync();
+}
+
+@DefaultTask()
+@Depends(test, copyright)
+build() {
+  Pub.build();
+}
+
+@Task()
+clean() => defaultClean();
+
+@Task()
+generate() {
+  Pub.run('build_runner', arguments: ['build']);
+}
+
+const _copyright =
+    '''// Copyright 2020, the Flutter project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.''';
+
+@Task()
+copyright() async {
+  var files = <File>[];
+  await for (var file in _filesWithoutCopyright()) {
+    files.add(file);
+  }
+
+  if (files.isNotEmpty) {
+    log('Found Dart files without a copyright header:');
+    for (var file in files) log(file.toString());
+    fail('run "grind fix-copyright" to add copyright headers');
+  }
+}
+
+@Task()
+fixCopyright() async {
+  await for (var file in _filesWithoutCopyright()) {
+    var contents = await file.readAsString();
+    file.writeAsString(_copyright + '\n\n' + contents);
+  }
+}
+
+Stream<File> _filesWithoutCopyright() async* {
```

Comment:
You should make a build_runner plugin out of this and publish it for use in the rest of our projects. 😆

Reply:
Sounds good, I think I'll follow up with that change in a separate PR